src/Umbraco.Core/Persistence/Repositories/VersionableRepositoryBase.cs

```diff
@@ -286,30 +286,47 @@ private Sql GetSortedSqlForPagedResults(Sql sql, Direction orderDirection, strin
             }
             else
             {
-                // Sorting by a custom field, so set-up sub-query for ORDER BY clause to pull through valie
+                // Sorting by a custom field, so set-up sub-query for ORDER BY clause to pull through value
                 // from most recent content version for the given order by field
                 var sortedInt = string.Format(SqlSyntax.ConvertIntegerToOrderableString, "dataInt");
                 var sortedDate = string.Format(SqlSyntax.ConvertDateToOrderableString, "dataDate");
                 var sortedString = string.Format("COALESCE({0},'')", "dataNvarchar");
                 var sortedDecimal = string.Format(SqlSyntax.ConvertDecimalToOrderableString, "dataDecimal");
 
+                //these are defaults that will be used in the query - they can be overridden for non-versioned entities or document entities

```

Comment:
@clausjensen, @zpqrtbnk is going to have heart attack merging these changes/fixes into v8  :smile: